### PR TITLE
Model submission for Kloubak sensor configuration 3 by Robotika

### PR DESCRIPTION
**Pull request by m3d**
This is extended version of Robotika robot "Kloubak" planned for System Finals:
kloubak-2021-03-30
There are two new RGBD/RealSense cameras (4 total) better covering the ground surface.

### CODE REVIEW

**Comment by m3d** [comment]
p.s. note, that this configuration is already with breadcrumbs

**Review by acschang — COMMENTED** [review]
The initial assessment of the model is complete and no changes are currently warranted. This submission will continue to be evaluated in the next stage of our review process.

File: submitted_models/robotika_kloubak_sensor_config_3/model.config [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  <version>1.0</version>
+  <sdf version='1.6'>model.sdf</sdf>
+
+  <author>
+    <name>Martin Dlouhy</name>
+    <email>md@robotika.cz</email>
+  </author>
+
+  <description>
+    robot Kloubak - articulated robot, symmetric configuration, two planar lidars, two RGB cameras and two depth cameras.
```
[the remark this answers is not in the document]
Comment (acschang) [reply submitted_models/robotika_kloubak_sensor_config_3/model.config:14]: Fixed in https://github.com/osrf/subt/pull/914/commits/429768a6d7c57c2b462ddd5b52f153fd8d48238d

File: submitted_models/robotika_kloubak_sensor_config_3/launch/vehicle_topics.launch [hunk 2]
```diff
@@ -0,0 +1,393 @@
...
+      </node>
+      <node
+        pkg="ros_ign_bridge"
+        type="parameter_bridge"
+        name="ros_ign_bridge_breadcrumbs_remaining"
+        args="/model/$(arg name)/breadcrumb/deploy/remaining@std_msgs/Int32[ignition.msgs.Int32">
+        <remap from="/model/$(arg name)/breadcrumb/deploy/remaining" to="breadcrumb/remaining"/>
+      </node>
+    </group>
+  </group>
```
[the remark this answers is not in the document]
Comment (acschang) [reply submitted_models/robotika_kloubak_sensor_config_3/launch/vehicle_topics.launch:392]: Fixed in https://github.com/osrf/subt/pull/914/commits/db52d78e7d55f34ba60d139495003d7b6582aea7

File: submitted_models/robotika_kloubak_sensor_config_3/model.sdf [hunk 3]
```diff
@@ -0,0 +1,1176 @@
...
+            <update_rate>10</update_rate>
+            <camera name="camera_front">
+                <horizontal_fov>2.27</horizontal_fov>
+                <lens>
+                    <intrinsics>
+                        <!-- fx = fy = width / ( 2 * tan (hfov / 2 ) ) -->
+                        <fx>149.01</fx>
+                        <fy>149.01</fy>
+                        <!-- cx = ( width - 1 ) / 2 -->
+                        <cx>339.5</cx>
```
[the remark this answers is not in the document]
Comment (acschang) [reply submitted_models/robotika_kloubak_sensor_config_3/model.sdf:320]: Fixed in https://github.com/osrf/subt/pull/914/commits/df7aa86d4c925992de171d3bbc3f5326ae248e6b

File: submitted_models/robotika_kloubak_sensor_config_3/model.sdf [hunk 4]
```diff
@@ -0,0 +1,1176 @@
...
+            <update_rate>10</update_rate>
+            <camera name="camera_back">
+                <horizontal_fov>2.27</horizontal_fov>
+                <lens>
+                    <intrinsics>
+                        <!-- fx = fy = width / ( 2 * tan (hfov / 2 ) ) -->
+                        <fx>149.01</fx>
+                        <fy>149.01</fy>
+                        <!-- cx = ( width - 1 ) / 2 -->
+                        <cx>339.5</cx>
```
[the remark this answers is not in the document]
Comment (acschang) [reply submitted_models/robotika_kloubak_sensor_config_3/model.sdf:727]: Fixed in https://github.com/osrf/subt/pull/914/commits/df7aa86d4c925992de171d3bbc3f5326ae248e6b

**Pull request closed without merging**